test/astarte_core/mapping/value_type_test.exs

```diff
@@ -19,8 +19,15 @@ defmodule Astarte.Core.Mapping.ValueTypeTest do
     assert ValueType.validate_value(:boolean, false) == :ok
 
     assert ValueType.validate_value(:binaryblob, <<0, 1, 2, 3, 4>>) == :ok
+
+    # TODO: remove this when all dependents on this library move to Cyanide 2.0
```

Comment:
Did you check this is not already the case? I guess most of the users of `astarte_core` inherit Cyanide from it